V3 updates (#54)
INFURA/documentation: docs/README.md

Getting started with Infura takes just a few minutes once you’ve [signed up](https://infura.io/register).

> After sign up, you will create your first "Project", which will include a v3 "Project ID" to use within your app when making requests to infura.io. This project id also provides customized insights and analytics, and routes your data where it needs to go to access the most powerful infrastructure for Ethereum. [Learn more about getting started](/docs/gettingStarted/chooseaNetwork). If you have any issues check our other resources below for help.

Reviewer: minor: comma after 'issues' -> If you have any issues, check

Author: Fixed

> After sign up, you will create your first "Project", which will include a v3 "Project ID" to use within your app when making requests to infura.io. This project id also provides customized insights and analytics, and routes your data where it needs to go to access the most powerful infrastructure for Ethereum. [Learn more about getting started](/docs/gettingStarted/chooseaNetwork). If you have any issues check our other resources below for help.

Author: ```suggestion
After sign up, you will create your first "Project", which will include a v3 "Project ID" to use within your app when making requests to infura.io. This project id also provides customized insights and analytics, and routes your data where it needs to go to access the most powerful infrastructure for Ethereum. [Learn more about getting started](/docs/gettingStarted/chooseaNetwork). If you have any issues, check our other resources below for help.
```

> After sign up, you will create your first "Project", which will include a v3 "Project ID" to use within your app when making requests to infura.io. This project id also provides customized insights and analytics, and routes your data where it needs to go to access the most powerful infrastructure for Ethereum. [Learn more about getting started](/docs/gettingStarted/chooseaNetwork). If you have any issues, check our other resources below for help.

Reviewer 2: `project id` should probably be `project ID` for consistency.

Author: Fixed in db0549a

## Ethereum API

> Seamlessly access Ethereum via the Infura load-balanced nodes and smart architecture the same way you would via your own nodes. We have built services and APIs around [JSON RPC](https://github.com/ethereum/wiki/wiki/JSON-RPC) over both HTTPS and WebSockets that you can use with your favorite libraries and frameworks, on four Ethereum networks. [Read more about our Ethereum API](/docs/gettingStarted/makeRequests).

Reviewer 3: ```suggestion
Seamlessly access Ethereum via the Infura API the same way you would via your own nodes. We have built the Infura API and services around [JSON RPC](https://github.com/ethereum/wiki/wiki/JSON-RPC) over both HTTPS and WebSockets that you can use with your favorite libraries and frameworks, on four Ethereum networks. [Read more about our Ethereum API](/docs/gettingStarted/makeRequests).
```

> ## IPFS API

Reviewer 2: ```suggestion
## WEBSOCKETS API
```

> Access the [IPFS API](https://ipfs.io/docs/api/) and our [public IPFS gateway](https://ipfs.infura.io/ipfs/Qmaisz6NMhDB51cCvNWa1GMS7LU1pAxdF4Ld6Ft9kZEP2a) over HTTPS like you would be able to via your own IPFS nodes. [Read more about our IPFS API](/docs/ipfs/get/block_get).

Reviewer 2: ```suggestion
Use our JSON-RPC websockets transport to create stateful subscriptions for new blocks. Access the [Ethereum Pub/Sub API documentation](https://github.com/ethereum/go-ethereum/wiki/RPC-PUB-SUB) and [Parity Ethereum Pub/Sub API documentation](https://wiki.parity.io/JSONRPC-Parity-Pub-Sub-module.html). Read more about our [WebSocket API](/docs/ethereum/wss/introduction).
```

> ## WEBSOCKETS API

Reviewer 2: ```suggestion
## IPFS API
```

> Use our JSON-RPC websockets transport to create stateful subscriptions for new blocks. Access the [Ethereum Pub/Sub API documentation](https://github.com/ethereum/go-ethereum/wiki/RPC-PUB-SUB) and [Parity Ethereum Pub/Sub API documentation](https://wiki.parity.io/JSONRPC-Parity-Pub-Sub-module.html). Read more about our [WebSocket API](/docs/wss/introduction).

Reviewer 2: Since the rest of this PR clarifies that websockets are related to Ethereum only for now, perhaps this section should make Websockets API a subsection of Ethereum API, or at least move Websockets API after Ethereum API and before IPFS API.

Author: It was moved under ethereum, so this is probably a broken link, will verify and update.

Author: yup it 404s, updating

Author: Fixed in db0549a

Reviewer 2: I hadn't actually noticed the broken link, I was talking about moving the entire `## WEBSOCKETS API` block under `## Ethereum API`. I'll try to use the suggestion feature even if this may not be the best use case...

> Use our JSON-RPC websockets transport to create stateful subscriptions for new blocks. Access the [Ethereum Pub/Sub API documentation](https://github.com/ethereum/go-ethereum/wiki/RPC-PUB-SUB) and [Parity Ethereum Pub/Sub API documentation](https://wiki.parity.io/JSONRPC-Parity-Pub-Sub-module.html). Read more about our [WebSocket API](/docs/ethereum/wss/introduction).

Reviewer 2: ```suggestion
Access the [IPFS API](https://ipfs.io/docs/api/) and our [public IPFS gateway](https://ipfs.infura.io/ipfs/Qmaisz6NMhDB51cCvNWa1GMS7LU1pAxdF4Ld6Ft9kZEP2a) over HTTPS like you would be able to via your own IPFS nodes. [Read more about our IPFS API](/docs/ipfs/get/block_get).
```
